packages/shared/src/styles/responsive.module.scss

```diff
@@ -0,0 +1,103 @@
+/* -------------------------------------------------------------------------- */
+/* RESPONSIVE MIXINS                                                          */
+/* -------------------------------------------------------------------------- */
+
+// Variables ----------------------------------------------------------------
+$unit: 1px;
+$breakpoints: (
+    'mobile': 926,
+    'tablet': 1081,
+    'desktop': 1800,
```

Comment:
can you add a comment that the value in desktop is an upper limit and it's from 1082 to 1800?
Also, add comments for mobile and tablet. It's a bit confusing when you look at it 😅

Reply:
These variables are the `max-width` of the breakpoints, as these are used in the js methods. I add a comment to state this.

Reply:
Also, I've changed the desktop value to be more clear.